api/client/client.go

```diff
@@ -918,6 +919,11 @@ func (c *Client) PresenceServiceClient() presencepb.PresenceServiceClient {
 	return presencepb.NewPresenceServiceClient(c.conn)
 }
 
+// PresenceServiceClient returns an unadorned client for the presence service.
```

Comment:
```suggestion
// InventoryServiceClient returns an unadorned client for the inventory service.
```

Also, what does "unadorned client" mean?